values.yaml

```diff
@@ -127,31 +120,23 @@ pip:
   initialDelaySeconds: 300 # pip service takes a long time to start up
   downloadCommand: "./bin/download --admin-only"
   limits:
-    memory: 10Gi
-    cpu: 3
-    # cannot use a hyphen to match the k8s param due to https://github.com/helm/helm/issues/2192
-    ephemeral_storage: 35Gi
+    memory: 12Gi
```

Comment:
keep original. overriding values will be in another file that should not be committed to this repo